rb/lib/selenium/webdriver/remote/http/curb.rb

```diff
@@ -17,7 +17,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
-require 'curb'
+begin
```

Comment:
Thinking a bit more about this, we are changing behavior for end users. Previously, when the user attempted to use this file, it would fail with LoadError saying that `curb` was missing - it's a clear indication that the user needs to install `curb` manually. Now, it silently succeeds and fails later when the user attempts to use the HTTP client with a missing constant.

Let's revert this change and instead rescue LoadError inside `curb_spec.rb`.

Reply:
You have a good point there, I moved the rescue into the spec, and also added a raise in case it fails for the user to know what happened and what they can do next, thank you for the review!

Reply:
Sorry I might be nitpicking here, but why did you remove `require "curb"`? We want to load it so that when the user requires this HTTP client, they will see a load error that Curb is missing.

Reply:
I miss understood your previous comment, let me get it back in place, you are right, thanks